spynnaker/pyNN/models/neuron/master_pop_table.py

```diff
@@ -305,36 +368,33 @@ def extract_synaptic_matrix_data_location(
         # convert into a numpy arrays
```

Comment:
this method is redundant as the above caller is never called

Reply:
Removed